test/models/pawn_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class PawnTest < ActiveSupport::TestCase
```

Comment:
As stated, you still need some testing in place to verify can_capture_en_passant.  I think once you write and validate those tests you'll be all set....